analysis/org.eclipse.tracecompass.analysis.chromium.core/src/org/eclipse/tracecompass/internal/analysis/chromium/core/callstack/ChromiumCallStackProvider.java

```diff
@@ -92,20 +92,30 @@ protected boolean considerEvent(@NonNull ITmfEvent event) {
 
     @Override
     protected @Nullable ITmfStateValue functionEntry(@NonNull ITmfEvent event) {
-        if (event instanceof TraceEventEvent && 'B' == (((TraceEventEvent) event).getField().getPhase())) {
+        if (event instanceof TraceEventEvent && isEntry(event)) {
             return TmfStateValue.newValueString(event.getName());
         }
         return null;
     }
 
+    private static boolean isEntry(ITmfEvent event) {
+        char phase = ((TraceEventEvent) event).getField().getPhase();
```

Comment:
We might want to have a layout at some point: the goal is to have a UST JUL trace AND a Chromium trace behave exactly the same, so a ITraceEventTrace interface, with a TraceEventLayout. Or we should find a generic name for that kind of trace, such that ChromiumTrace is a <that kind of trace> and lttng ust jul trace also is. Though in the case of the LTTng UST, it may not be the trace itself, but the event that will implement the TraceEvent format, as the trace may contain other kinds of events.

Lots of potential thoughts for the future! 

Oh and I might want to benchmark my UI responsiveness analysis results with either a fileHandler or a LTTng UST handler.